### PR TITLE
CIR-1525-FullReturn1/3

### PR DESCRIPTION
**Pull request by AdamCoops**

### CODE REVIEW

File: public/api/conf/1.0/schemas/subSchemas/ultimateParent.json [hunk 1]
```diff
@@ -12,19 +12,19 @@
...
+      "description": "Ultimate parent's Corporation Tax Unique Taxpayer Reference.",
       "$ref": "../definitions/definitions.json#/definitions/utrType"
     },
     "sautr": {
       "description": "Ultimate parent company's self assessment Unique Taxpayer Reference.",
       "$ref": "../definitions/definitions.json#/definitions/utrType"
     },
     "countryOfIncorporation": {
-      "description": "Country of incorporation where the ultimate parent company is not a UK company. A valid ISO 3166-1 2 Alpha Country Code",
+      "description": "Country of incorporation where the ultimate parent company is not a UK company. A valid ISO 3166-1 2 Alpha Country Code.",
```
Comment (DevAndyroo) [inline public/api/conf/1.0/schemas/subSchemas/ultimateParent.json:27]: Wasn't one of the things @burns238 mentioned on Friday to not have punctuation at the end as it looks weird on the dev hub?

File: public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json [hunk 2]
```diff
@@ -690,7 +690,7 @@
             "additionalProperties": false
         },
         "submissionType": {
-            "description": "Indicate whether this is the first (original) return or an amended (revised) return",
+            "description": "Indicate if this is a revised return [this should be a boolean true or false]. Limited to the following possible values:",
```
Comment (DevAndyroo) [inline public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json:693]: I think we need this to be double checked, to me this doesn't make sense. Sounds like you're saying ti can be a boolean or the enum, right?

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json (hunk 2 above)
Comment (AdamCoops) [reply public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json:693]: Sorted this with Liam, we're gonna leave it as is whilst it gets checked and we'll raised and bug for it if it's wrong.

File: public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json [hunk 3]
```diff
@@ -690,7 +690,7 @@
             "additionalProperties": false
         },
         "submissionType": {
-            "description": "Indicate whether this is the first (original) return or an amended (revised) return",
+            "description": "Indicate if this is a revised return [this should be a boolean true or false]. Limited to the following possible values",
```
Comment (burns238) [inline public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json:693]: This isn't a boolean so I think we need to remove that bit in the square bracers. I also think the last sentence might be duplicated on the webpage after parsing

**Review by burns238 — CHANGES_REQUESTED** [review]

**Review by burns238 — APPROVED** [review]

**Pull request merged**